__TESTS__/unit/actions/Resize/AspectRatio.test.ts

```diff
@@ -0,0 +1,104 @@
+import getImageWithResize from "./shared/getImageWithResize";
+import crop from "../../../../src/actions/resize/ResizeActions/CropAction";
+import Gravity from "../../../../src/values/gravity/Gravity";
```

Comment:
For simplicity,
do you think we could just test that .aspectRatio(AspectRatio.AR1X1()) is returning the right string?
do we really need all the boilerplate of the image?